api/checks/runs.go

```diff
@@ -86,16 +87,33 @@ func getExistingCheckRuns(ctx context.Context, suite shared.CheckSuite) ([]*gith
 		return nil, err
 	}
 
+	var runs []*github.CheckRun
 	options := github.ListCheckRunsOptions {
 		ListOptions: github.ListOptions {
+			// 100 is the maximum allowed items per page; see
+			// https://developer.github.com/v3/guides/traversing-with-pagination/#changing-the-number-of-items-received
 			PerPage: 100,
 		},
 	}
-	runs, _, err := client.Checks.ListCheckRunsForRef(ctx, suite.Owner, suite.Repo, suite.SHA, &options)
-	if err != nil {
-		return nil, err
+
+	// As a safety-check, we will not do more than 5 iterations (at 100
+	// check runs per page, this gives us a 500 run upper limit).
+	for i := 0; i < 5; i++ {
```

Comment:
This is may be not enough, 0f3f2b950d26ab0c88084c1537143e38712cb4b9 has 227 check_runs and it would have more if `epochs/daily` wasn't currently failing due to the sink-task dependency problem.

Reply:
http get https://api.github.com/repos/web-platform-tests/wpt/commits/0f3f2b950d26ab0c88084c1537143e38712cb4b9/check-runs Accept:application/vnd.github.antiope-preview+json, FTR

Reply:
I'd opt to double the limit; let's give it 10 pages.